src/plugins/intel_cpu/src/nodes/input.cpp

```diff
@@ -227,8 +227,7 @@ Input::Input(const std::shared_ptr<ov::Node>& op, const GraphContext::CPtr conte
                                        op->get_type_name(),
                                        " with name ",
                                        op->get_friendly_name());
-
-    constant = ConstantType::NoConst;
+    constant = ConstantType::StrictNoConst;
```

Comment:
@dmitry-gorokhov @v-Golubev from my understanding if it is not ngraph Constant, then it is always StrictNoConst

Reply:
Acually no. Example: we are moving some node from activations path to constant subgraph (like mamtul weights). In that case it should become `Const`.

Reply:
Discussed, the change is correct.